index.js

```diff
@@ -1,23 +1,24 @@
-// require the Twilio module and RequestClient
-const twilio = require('twilio');
-const MyRequestClient = require('./MyRequestClient');
+// require the Twilio module and MyRequestClient
+const twilio = require("twilio");
+const MyRequestClient = require("./MyRequestClient");
 
 // Load environment variables
-require('dotenv').config()
+require("dotenv").config();
```

Comment:
nit: this was an issue with the original sample; it's best practice to have dotenv be the first line of the entry point to the project instead of down here, just in case a required dependency has env vars initialized in time

Reply:
Thank you. I'll fix this.